File: Backend/src/EventScheduler/EventScheduler.Domain/Speaker.cs
﻿using System.Collections.Generic;
namespace EventScheduler.Domain
{
    public class Speaker
    {
        public int Id { get; set; }
        public string MiniResume { get; set; }
        public string ImageURL { get; set; }
        public string Phone { get; set; }
        public string Email{ get; set; }
        public IEnumerable<SocialNetwork> RedesSociais { get; set; }
        public IEnumerable<SpeakerEvent> PalestrantesEventos { get; set; }
    }
}

File: Frontend/EventScheduler/src/app/models/Speaker.ts
import { SocialNetwork } from "./SocialNetwork";
import { SpeakerEvent } from "./SpeakerEvent";

export interface Speaker {
    id : number;
    name : string;
    miniResume : string;
    imageURL : string;
    phone : string;
    email : string;
    socialNetworks : SocialNetwork[];
    palestrantesEventos : SpeakerEvent[];
}
